labs/lab9/9.md

```diff
@@ -0,0 +1,250 @@
+---
+title: Lab 9 - Web Application
+layout: lab
+---
+
+## Overview
+
+We’re going to set up a flask app. And we are going to call it basicapp
+
+There is 3 main component to this lab. We will install Nginx, uWSGI, and flask through the apt command. Then we will git clone an example app and config files from class repository. Next, we will modify the configuration files. If configured correctly, we'll go through the startup procedure. Then end it with some debugging tips and tricks. 
+
+## Note
+
+* No sudo was included in any command examples. Use your best judgment to include sudo or not. 
+
+### The Setup
+
+We need virtualenv and git for staging. Run
```

Comment:
I'd explain what virtualenv is up here, and instead of giving them the `apt-get` command directly, just tell them to install it as we've already taught them how. Also, git is already installed everywhere through puppet: https://github.com/0xcf/decal-puppet/blob/c4247058aecb95e3d9181226eb62001ac89d8e10/modules/decal_common/manifests/init.pp#L18